src/index.ts

```diff
@@ -737,6 +762,14 @@ class NixInstallerAction extends DetSysAction {
         }
       }
 
+      const plausibleDeterminateOptions = [];
+      const plausibleDeterminateArguments = [];
+      if (this.determinate) {
+        plausibleDeterminateOptions.push("--entrypoint");
+        plausibleDeterminateOptions.push("/usr/local/bin/determinate-nixd");
+        plausibleDeterminateArguments.push("daemon");
```

Comment:
Hm, consider, do we want to update the default entrypoint for the image? I could see arguments both ways.

Reply:
Not sure what the "recommit the tar.gz" means, presumably it's just another nix build artifact? I guess you're meaning rev wherever we publish the container / IDS?

I am not worried about it, just wanted to mention. Dropping seems good/fine too. Whenever.

Reply:
Unfortunately since it needs to be available before Nix can run, it is literally checked in: https://github.com/DeterminateSystems/nix-installer-action/tree/main/docker-shim